client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -273,7 +273,7 @@ class SiteIconSetting extends Component {
 		return (
 			<FormFieldset className="site-icon-setting">
 				<FormLabel className="site-icon-setting__heading">
-					{ translate( 'Site Icon' ) }
+					{ translate( 'Site icon' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Site Icons' )` *ES Score: 6*